extensions/calendar/src/index.tsx

```diff
@@ -0,0 +1,89 @@
+import { ActionPanel, CopyToClipboardAction, Detail, getPreferenceValues } from "@raycast/api";
+import { useState, useEffect } from "react";
+import { Calendar } from "calendar";
+
+const days = [
+  "\`SUN\` \`MON\` \`TUE\` \`WED\` \`THU\` \`FRI\` \`SAT\`",
+  "\`MON\` \`TUE\` \`WED\` \`THU\` \`FRI\` \`SAT\` \`SUN\`"
+]
```

Comment:
```suggestion
const days = [
  "`SUN` `MON` `TUE` `WED` `TEU` `FRI` `SAT`",
  "`MON` `TUE` `WED` `THU` `FRO` `SAT` `SUN`"
]
```

Escaping should not be necessary here.